recipes/vulkan-validationlayers/all/CMakeLists.txt

```diff
@@ -4,10 +4,11 @@ project(cmake_wrapper)
 include(conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
-add_library(glslang INTERFACE)
-target_link_libraries(glslang INTERFACE CONAN_PKG::glslang)
+add_library(glslang INTERFACE) # fake target for upstream CMakeLists (glslang required by tests only)
```

Comment:
I really like this more then patches

Reply:
At least, it works for any version. Patches are pretty limited